docs/management/connectors/action-types/opsgenie.asciidoc

```diff
@@ -0,0 +1,173 @@
+[role="xpack"]
+[[opsgenie-action-type]]
+=== Opsgenie connector and action
+++++
+<titleabbrev>Opsgenie</titleabbrev>
+++++
+
+The Opsgenie connector uses the https://docs.opsgenie.com/docs/alert-api[Opsgenie Alert API].
+
+[float]
+[[opsgenie-connector-configuration]]
+==== Connector configuration
+
+Opsgenie connectors have the following configuration properties.
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+URL::     The Opsgenie URL: https://api.opsgenie.com. For the EU instance of Opsgenie use: https://api.eu.opsgenie.com. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
+API Key::   The Opsgenie API authentication key for HTTP Basic authentication. Refer to https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[How to generate an Opsgenie API key] for more information.
```

Comment:
IMO we should make it clearer that they're leaving our documentation with this link:
```suggestion
API Key::   The Opsgenie API authentication key for HTTP Basic authentication. For more details about generating Opsgenie API keys, refer to https://support.atlassian.com/opsgenie/docs/create-a-default-api-integration/[Opsgenie documentation].
```